src/commands/general/config/reset.ts

```diff
@@ -0,0 +1,70 @@
+import { Command, RegisterSubCommand } from '@kaname-png/plugin-subcommands-advanced';
+import { Result } from '@sapphire/result';
+import generateEmbed from '../../../helpers/generate/embed';
+import { setDefaultConfig } from '../../../helpers/provide/config';
+import { SUCCESS, ARROW_RIGHT } from '../../../helpers/provide/environment';
+import replyToInteraction from '../../../helpers/send/response';
+import { type ConfigName, configNameExtended } from '../../../lib/database';
+import thinking from '../../../lib/discord/thinking';
+
+@RegisterSubCommand('config', (builder) =>
+	builder
+		.setName('reset')
+		.setDescription('Reset settings from your server config')
+		.addStringOption((option) =>
+			option
+				.setName('config')
+				.setDescription('Config that you want to remove')
+				.addChoices(
+					{
+						name: 'Announcement Channel',
+						value: 'announcementChannel',
+					},
+					{
+						name: 'Overview Channel',
+						value: 'overviewChannel',
+					},
+					{
+						name: 'Birthday Role',
+						value: 'birthdayRole',
+					},
+					{
+						name: 'Ping Role',
+						value: 'birthdayPingRole',
+					},
+					{
+						name: 'Announcement Message',
+						value: 'announcementMessage',
+					},
+					// {
+					// 	name: 'Logs',
+					// 	value: 'logs'
+					// }
+				)
+				.setRequired(true),
+		),
+)

```

Comment:
Please outsource this into a separate file in the `lib/commands/config` file if possible